### PR TITLE
[BTAT-7560] - different content for user with digital preference and verified email

### PR DESCRIPTION
**Pull request by sraspin**
# Pull Request Checklist
* [x] Branch is rebased with master
* [x] Added Unit Tests for changed functionality
* [x] Ran `sbt clean compile coverage test it:test coverageReport`, all tests pass and coverage meets minimum
* [x] Checked libraries are up to date with `sbt validate`. (If applicable) library updates are done in a separate, single commit
* [x] Ran [change-vat-acceptance-tests](https://github.com/hmrc/change-vat-acceptance-tests) (see README of repo)
* [ ] Ran [change-vat-performance-tests](https://github.com/hmrc/change-vat-performance-tests) (see README of repo)
* [ ] (If applicable) raised app-config-* PRs
* [ ] (If applicable) ran Wave test for changes to views
* [ ] (If applicable) added Integration Test

### CODE REVIEW

File: app/controllers/returnFrequency/ChangeReturnFrequencyConfirmation.scala [hunk 1]
```diff
@@ -54,15 +54,20 @@ class ChangeReturnFrequencyConfirmation @Inject()(val messagesApi: MessagesApi,
...
+      cPref <- contactPreferenceService.getContactPreference(user.vrn).map {
+        case Right(contact) =>
+          auditService.extendedAudit(
+            ContactPreferenceAuditModel(user.vrn, contact.preference),
+            Some(controllers.returnFrequency.routes.ChangeReturnFrequencyConfirmation.show("non-agent").url)
+          )
+          Some(contact.preference)
+        case _ => None
+      }
+      verifiedEmail <- customerCircumstanceDetailsService.getCustomerCircumstanceDetails(user.vrn).map {
```
Comment (miranda-hawkes) [inline app/controllers/returnFrequency/ChangeReturnFrequencyConfirmation.scala]: See https://github.com/hmrc/manage-vat-subscription-frontend/pull/361#discussion_r383798290

File: test/controllers/returnFrequency/ChangeReturnFrequencyConfirmationSpec.scala [hunk 2]
```diff
@@ -101,6 +101,7 @@ class ChangeReturnFrequencyConfirmationSpec extends BaseSpec
           lazy val result = {
             mockAuthorise(mtdVatAuthorisedResponse)
             mockContactPreferenceSuccess(ContactPreference("DIGITAL"))
+            mockCustomerDetailsSuccess(circumstanceDetailsModelMax)
```
Comment (miranda-hawkes) [inline test/controllers/returnFrequency/ChangeReturnFrequencyConfirmationSpec.scala]: Need to add some tests for new logic in controller as in manage-vat PR, i.e. success vs fail vs null emailVerified boolean

**Comment by sraspin** [comment]
https://github.com/hmrc/app-config-base/pull/3141
https://github.com/hmrc/app-config-production/pull/5907

File: app/config/features/Features.scala [hunk 3]
```diff
@@ -25,4 +25,5 @@ class Features @Inject()(implicit config: Configuration) {
 
   val stubAgentClientLookup = new Feature(ConfigKeys.stubAgentClientLookupFeature)
   val stubContactPreferencesFeature = new Feature(ConfigKeys.stubContactPreferencesFeature)
+  val emailVerifiedFeature = new Feature(ConfigKeys.emailVerifiedFeature)
```
Comment (miranda-hawkes) [inline app/config/features/Features.scala]: See https://github.com/hmrc/manage-vat-subscription-frontend/pull/361#discussion_r384577310

File: build.sbt [hunk 4]
```diff
@@ -54,11 +54,11 @@ lazy val coverageSettings: Seq[Setting[_]] = {
 
 val compile = Seq(
   play.sbt.PlayImport.ws,
-  "uk.gov.hmrc" %% "govuk-template" % "5.48.0-play-25",
-  "uk.gov.hmrc" %% "play-ui" % "8.6.0-play-25",
+  "uk.gov.hmrc" %% "govuk-template" % "5.52.0-play-25",
```
Comment (miranda-hawkes) [inline build.sbt]: sbt-artifactory can also be upgraded in plugins.sbt

File: test/views/returnFrequency/ChangeReturnFrequencyConfirmationViewSpec.scala [hunk 5]
```diff
@@ -26,7 +26,90 @@ class ChangeReturnFrequencyConfirmationViewSpec extends ViewBaseSpec {
...
+
+          "has link back to customer details page" in {
+            element("#finish").attr("href") shouldBe mockAppConfig.manageVatUrl
+          }
+        }
+      }
+
+      "emailVerified is false" should {
+
+        lazy val view = views.html.returnFrequency.change_return_frequency_confirmation(contactPref = Some("DIGITAL"))(user, messages, mockAppConfig)
```
Comment (miranda-hawkes) [inline test/views/returnFrequency/ChangeReturnFrequencyConfirmationViewSpec.scala]: As the only thing changing between this and the above scenario is the email preference, I would bother retesting all the other features of the page - only the content regarding their preferences

**Pull request merged**